samcli/local/common/runtime_template.py

```diff
@@ -49,7 +49,7 @@
             "runtimes": ["dotnetcore", "dotnetcore1.0", "dotnetcore2.0", "dotnetcore2.1"],
             "dependency_manager": "cli-package",
             "init_location": os.path.join(_templates, "cookiecutter-aws-sam-hello-dotnet"),
-            "build": False
+            "build": True
```

Comment:
Nit: this indicts .net core 1.0 is also supported with build. Is that true? The integ tests only have 2.0 and 2.1.

Reply:
Removing 1.0

Reply:
I actualy didnt remove it, since we use runtimes list to showcase available runtimes, we might need to add another field to showcase what the builds are for in the future.

Reply:
Makes sense